www/docs/server/error-formatting.md

```diff
@@ -7,9 +7,9 @@ slug: /server/error-formatting
 
 The error formatting in your router will be inferred all the way to your client (&&nbsp;React&nbsp;components)
 
-## Usage example highlighted
+## Usage example-zod validation error
```

Comment:
not sure about these header changes?

maybe `## Example 1: Zod Errors` (and similar for prisma one below)

Reply:
```suggestion
## Usage example: Zod validation error
```